config/cesm/config_grids.xml

```diff
@@ -120,6 +120,15 @@
       <grid name="rof">null</grid>
     </model_grid>
 
+    <!-- Regional NLDAS grid over the U.S. (roughly 12km resolution), with the standard NLDAS mask -->
+    <model_grid alias="nldas_mnldas" compset="DATM.+CLM">
```

Comment:
Does this seem like a reasonable name for the alias? I struggled a bit, because the mask has the same name as the grid: we're using the standard NLDAS mask with the NLDAS grid.

For many grid aliases, we also give the ocean resolution, but that seems irrelevant here, and would lead to even more redundancy in the name (`nldas_nldas_mnldas`).

But I'm very open to suggestions here to keep this consistent with any grid naming conventions.

Reply:
why nldas instead of 12km or nl12?

Reply:
Good idea, @jedwards4b . Though a couple of thoughts on this are:
- I prefer spelling things out a bit more, so would probably go with `nldas12_mnldas` or `nldas12km_mnldas`
- Do you think that including `12` or `12km` in the name gives the impression that the grid is at exactly 12 km (which isn't true)? This grid is actually at 1/8 deg – so roughly, but not exactly, 12 km.

@barlage I'd also welcome any opinions you have on this.

Reply:
@billsacks My preference would be to keep this as `nldas_mnldas` and leave the details in the grid definition below; NLDAS is a well-defined grid but another consideration would be to add an NLDAS version here (e.g. nldas2) since I believe NLDAS v3 may have a different, higher resolution grid when released. Is there any need to explicitly declare this as regional in the model_grid?